I have added a commands to Export proxy variables for clear understan… (#80848)
openshift/openshift-docs: modules/backup-etcd.adoc

sh-4.4# chroot /host
----

. If the cluster-wide proxy is enabled, ensure that you exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
+
.. To export the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables, run the following commands:

> 

Reviewer: I think the formatting issues will be resolved if you remove this line. For example, the text should look like this:

```
.. To export the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables, run the following commands:
+
[source,terminal]
----
$ export HTTP_PROXY=http://<your_proxy.example.com>:8080
----
+
[source,terminal]
----
$ export HTTPS_PROXY=https://<your_proxy.example.com>:8080
----
+
[source,terminal]
----
$ export NO_PROXY=<example.com>
----

> . If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.

Reviewer: ```suggestion
. If the cluster-wide proxy is enabled, ensure that you exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
```

> 

Reviewer: ```suggestion
+
```

> [TIP]

Reviewer: ```suggestion

```

> ===

Reviewer: ```suggestion

```

> Use following Commands to export `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY`  environment variables.

Reviewer: ```suggestion
.. To export the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables, run the following commands:
```

> export HTTP_PROXY=http://your-proxy.example.com:8080

Reviewer: ```suggestion
+
[source,terminal]
----
$ export HTTP_PROXY=http://<your_proxy.example.com>:8080
----
```

> 

Reviewer: ```suggestion
+
```

> export HTTPS_PROXY=https://your-proxy.example.com:8080

Reviewer: ```suggestion
[source,terminal]
----
$ export HTTPS_PROXY=https://<your_proxy.example.com>:8080
----
```

> 

Reviewer: ```suggestion
+
```

> export NO_PROXY=example.com

Reviewer: ```suggestion
[source,terminal]
----
$ export NO_PROXY=<example.com>
----
```

> ===

Reviewer: ```suggestion

```
